source/frontend/src/features/leases/detail/LeasePages/details/LeaseDetailsForm.tsx

```diff
@@ -12,21 +12,19 @@ import styled from 'styled-components';
 
 import DetailDocumentation from './DetailDocumentation';
 
-export interface IDetailsProps {
-  isEditing?: boolean;
-}
+export interface IDetailsProps {}
 
-export const LeaseDetailsForm: React.FunctionComponent<React.PropsWithChildren<IDetailsProps>> = ({
-  isEditing,
-}) => {
+export const LeaseDetailsForm: React.FunctionComponent<
+  React.PropsWithChildren<IDetailsProps>
+> = () => {
   const { lease } = React.useContext(LeaseStateContext);
   return (
     <Formik initialValues={{ ...defaultLease, ...lease }} enableReinitialize={true} onSubmit={noop}>
       <StyledDetails>
         <DetailTermInformation />
-        <PropertiesInformation disabled={!isEditing} />
-        <DetailAdministration disabled={!isEditing} />
-        <DetailDocumentation disabled={!isEditing} />
+        <PropertiesInformation disabled={true} />
+        <DetailAdministration disabled={true} />
+        <DetailDocumentation disabled={true} />
```

Comment:
This needs to be refactored given the underlying components are only used for view